first_order_model/shrink_util.py

```diff
@@ -1061,52 +1126,54 @@ def shrink_model(model_copy, layer_graph, layer, count, sort):
         model_copy = channel_prune(model_copy, deletions)
```

Comment:
This loop needs more clarification. Seems to be going through deletions in order and deleting the relevant filters, but please rename your variables to be more descriptive and add a loop comment

Reply:
Should be improved